blocks/article-list/article-list.js

```diff
@@ -1,117 +1,43 @@
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                    
-                    <div
-                        class="grid__structure featured-articles-with-teaser__items featured-articles-with-teaser__items--${articles.length}">
-                        ${articles.map((article) => [article, placeholders]).map(articleTemplate).join('')}
-                        
+  let articlesHTML = '';
+  for (let i = 0; i < articles.length; i += 1) {
+    articlesHTML += getArticle(articles[i], placeholders, i);
+  }
 
-                        <div
-                            class="grid__column featured-articles-with-teaser__item featured-articles-with-teaser__item--teaser">
-                            <div class="featured-articles-with-teaser__teaser">
-                                <div class="headline hl-s featured-articles-with-teaser__teaser-headline">
-                                    <span>
-                                        <h3>
-                                            <span class="headline__main">${placeholders.furtherarticlesheadline}</span>
-                                        </h3>
-                                    </span>
-                                </div>
-                                <div class="text text--body-m featured-articles-with-teaser__teaser-text">
-                                    <p>${placeholders.furtherarticlesteaser}<br>
-                                    </p>
-                                </div>
-                                <div class="featured-articles-with-teaser__teaser-link">
-                                    <div class="button-link button-link--link">
-                                        <a class=" button-link--icon  "
-                                            href="${placeholders.furtherarticleshref}"
-                                            target="_blank">
-                                            <span class="button-link__content">
-                                                <span>
-                                                    <span>${placeholders.furtherarticlesbutton}</span>
-                                                </span>
-                                                <span class="icon icon-external-link icon--symbol">
-                                                </span>
-                                            </span>
-                                        </a>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-                <div class="grid__content-bg--full featured-articles-with-teaser__grid-background"></div>
-            </div>
-        </div>
+  return `
+    <div class="article-list-items">
```

Comment:
If we remove the `article-list-items` div and also the exception for `article-list` [from here](https://github.com/hlxsites/zeiss/blob/ea20c954e14e4de68e28ef1f20dba78ee5b6e677/styles/simplified-styles.css#L151-L159) then the `block` itself could provide the `grid` layout. The advantage would be that this `article-list` block would inherit the global styles applicable for all the blocks thus providing a consistent styling for all the blocks in terms of margins, paddings etc. The additional styling you have for `article-list` block itself can then be specified for the `.block.article-list` selector. wdyt?

Reply:
Hi @sdmcraft I think its a great idea, and should simplify things further.

One issue with it though is that the `:not():not():not():not()...` block has super-high priority which makes overriding elements from it in specific elements only possible with `!important` which I don't think is great. 

In the screenshot below you can see that the specific `.block.article-list` comes after the `:not:not...` definition. I've used `!important` for now, would you have a better solution?

<img width="608" alt="Screenshot 2023-05-15 at 11 09 28" src="https://github.com/hlxsites/zeiss/assets/226514/7e0dc113-f1cf-4cfa-b0d6-4e6b42585826">

Reply:
Ah I remember you mentioned about this earlier as well. I am hoping that the list of those `:not` selectors should no longer be needed now since most of the blocks are simplified now. For now, let's just have that `!important` temporarily.